sql/world/uai_WorldSpellDBC.sql

```diff
@@ -0,0 +1,8 @@
+use world;
```

Comment:
```suggestion
```
Do not put the "use db" in SQL files, most people will have a different DB name than world :)

Reply:
I would like to comment it out with a -- use world;  Ok?